wpilib/wpilib/digitaloutput.py

```diff
@@ -138,8 +147,10 @@ def disablePWM(self):
 
         Free up one of the 6 DO PWM generator resources that were in use.
         """
-        if self.pwmGenerator is None:
+        if self.pwmGenerator is not self.invalidPwmGenerator:
             return
+        hal.setDigitalPWMOutputChannel(self.pwmGenerator, self.kDigitalChannels)
```

Comment:
I guess I shouldn't be swapping between `self._pwmGenerator` and `self.pwmGenerator`